content/en/docs/spinnaker/terraform-enable-integration.md

```diff
@@ -404,6 +438,19 @@ Use the `static` credential kind to provide any arbitrary key/value pair that is
       value: us-west-2
 ```
 
+**Terraform remote backend**
+
+Use the `tfc` credential kind to provide authentication to remote Terraform backends.
+
+```yaml
+- name: milton-tfc # Unique name for the profile. Shows up in Deck.
+  variables:
+  - kind: tfc
+    options: 
```

Comment:
@aelath what are the actual values for the options

Reply:
```
        options:
          domain: app.terraform.io # or terraform enterprise URL
          token: authentication-token
```